x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plan/logical/Pivot.java

```diff
@@ -47,7 +48,20 @@ public Pivot(Source source, LogicalPlan child, Expression column, List<NamedExpr
 
     @Override
     protected Pivot replaceChild(LogicalPlan newChild) {
-        return new Pivot(source(), newChild, column, values, aggregates);
+
```

Comment:
Could you please add some comment on why this is needed?

Reply:
done